app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -446,6 +446,14 @@ input[type="url"]:-ms-input-placeholder{
         margin-bottom: $base-margin;
       }
     }
+
+    /* Prevents jogging of an input's placeholder text due to safari's form autofill. */
+    & input{
```

Comment:
I don't think you need the & here

& input {
}

and 

input {
}

Would be the same thing. I also wonder if we're better off putting this on all form tags? So like 
form input,
form select{
  height: auto;
}